account_budget_oca/views/res_config_settings_views.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="res_config_settings_view_form" model="ir.ui.view">
+        <field name="name">res.config.settings.view.form.inherit.account.budget</field>
+        <field name="model">res.config.settings</field>
+        <field name="inherit_id" ref="account.res_config_settings_view_form"/>
+        <field name="arch" type="xml">
+            <div id="msg_account_budget" position="replace">
+                <div class="content-group" attrs="{'invisible': [('module_account_budget', '=', False)]}">
```

Comment:
Should be `attrs="{'invisible': [('module_account_budget_oca', '=', False)]}"`.

Field `module_account_budget_oca` should be added: `<field name="module_account_budget_oca"/>`

And [this field](https://github.com/odoo/odoo/blob/12.0/addons/account/views/res_config_settings_views.xml#L413) should be set to `invisible="1"`.

Reply:
I didn't do exactly this, as module_account_budget is a field in really that is in account module and it installs module account_budget, I have just made it visible the link to budgetary positions when you install the module account_budget_oca not check needed.

Reply:
Module `account_budget_oca` is alternative to `account_budget`, right?
So you should disable the installation of module `account_budget` in the settings interface, otherwise there will be a conflict. Just hiding field `module_account_budget` would be fine.

In the manifest you should also add:

```
    'excludes': ['account_budget'],
```

Reply:
I added the exclusion but hiding the field is not that simple, and after asking they told me not to worry about it.